_includes/manuals/3.1/1.2_release_notes.md

```diff
@@ -24,7 +24,293 @@ Existing installations should start thinking about a migration plan.
+* Edit button in experimental host page doesn't work \- [#33490](https://projects.theforeman.org/issues/33490)
+* Add setting for redirecting to the new host details page  \- [#33484](https://projects.theforeman.org/issues/33484)
+* "Login Page Footer Text" Blocking Login Button on Foreman UI \- [#33481](https://projects.theforeman.org/issues/33481)
+* Use the new confirm modal in the host status clear \- [#33473](https://projects.theforeman.org/issues/33473)
+* Host details page tabs re-render continuously \- [#33324](https://projects.theforeman.org/issues/33324)
+* broken Documentation button in search for react pages  \- [#33323](https://projects.theforeman.org/issues/33323)
+* Add power options dropdown to details card \- [#33268](https://projects.theforeman.org/issues/33268)
+* After creating new host group or Http proxy, page doesn't redirect to main landing page. \- [#33177](https://projects.theforeman.org/issues/33177)
+* Add console to the host details page \- [#32999](https://projects.theforeman.org/issues/32999)
+* Add manageBookmarks to pf4 bookmarks \- [#32656](https://projects.theforeman.org/issues/32656)
+* Move the host status page from foreman_wreckingball to core \- [#31638](https://projects.theforeman.org/issues/31638)
+* The time ago  creation line is not translated in the new host details page \- [#30984](https://projects.theforeman.org/issues/30984)
+
+#### Hammer
+* Remove a space in hammer's shebang, [#33810](http://projects.theforeman.org/issues/33810)
+* Wrap option descriptions to 80 chars, [#33129](http://projects.theforeman.org/issues/33129)
+* Don't store @context in field params, [#33259](http://projects.theforeman.org/issues/33259)
+* Change from superficial copy to deep copy of fields ([PR #348](https://github.com/theforeman/hammer-cli/pull/348)), [#29093](http://projects.theforeman.org/issues/29093)
+* Make api docs params to be the main options, [#33226](http://projects.theforeman.org/issues/33226)
+* Show depr warning only on option usage, [#33225](http://projects.theforeman.org/issues/33225)
+* Extract descs to option details section, [#32783](http://projects.theforeman.org/issues/32783)
+* Make sure provided options override defaults, [#33711](http://projects.theforeman.org/issues/33711)
+* Make template create/update commands use resolver, [#33721](http://projects.theforeman.org/issues/33721)
+* Revert fix rake version
+* Show status once for proxy info ([PR #581](https://github.com/theforeman/hammer-cli-foreman/pull/581)), [#19510](http://projects.theforeman.org/issues/19510)
+* Add proxy status and version to info command ([PR #572](https://github.com/theforeman/hammer-cli-foreman/pull/572))
+* Associate a vm via hammer to a compute resource, [#33047](http://projects.theforeman.org/issues/33047)
+* Fix option builders and tests, [#33226](http://projects.theforeman.org/issues/33226)
+* Add parent title option for hostgroup, [#32878](http://projects.theforeman.org/issues/32878)
```

Comment:
I suppose these were taken directly from hammer, but they seem to be formatted a bit differently?